math/minuit2/src/VariableMetricBuilder.cxx

```diff
@@ -106,10 +106,12 @@ FunctionMinimum VariableMetricBuilder::Minimum(const MnFcn &fcn, const GradientC
    int maxfcn_eff = maxfcn;
    int ipass = 0;
    bool iterate = false;
+   bool runHessian = false;
```

Comment:
```suggestion
   bool hessianComputed = false;
```

Reply:
It took me a while to understand what `runHessian` was doing, I think it is easier to understand with the name `hessianComputed`

Reply:
>  When using strategy 0 (i.e. when Hesse is not computed at the end of the minimization), the same state is added 2 times in the FunctionMinimum class.

Maybe this explanation could also be added as a code comment here?